WinUIGallery/Styles/CriticalButtonStyle.xaml

```diff
@@ -0,0 +1,191 @@
+                                 Color="#3FA0A0A0" />
+
+                <SolidColorBrush x:Key="CriticalButtonBorderBrush"
+                                 Color="#3FF1707B" />
+                <SolidColorBrush x:Key="CriticalButtonBorderBrushPointerOver"
+                                 Color="#3FE74856" />
+                <SolidColorBrush x:Key="CriticalButtonBorderBrushPressed"
+                                 Color="#3FA4262C" />
+                <SolidColorBrush x:Key="CriticalButtonBorderBrushDisabled"
+                                 Color="#3F3F3F3F" />
+
+                <SolidColorBrush x:Key="CriticalButtonForeground"
+                                 Color="#101010" />
+                <SolidColorBrush x:Key="CriticalButtonForegroundPointerOver"
+                                 Color="#101010" />
+                <SolidColorBrush x:Key="CriticalButtonForegroundPressed"
+                                 Color="#202020" />
+                <SolidColorBrush x:Key="CriticalButtonForegroundDisabled"
+                                 Color="#8A8A8A" />
+            </ResourceDictionary>
+
+            <!-- High Contrast -->
+            <ResourceDictionary x:Key="HighContrast">
+                <SolidColorBrush x:Key="CriticalButtonBackgroundPointerOver"
+                                 Color="Yellow" />
+                <SolidColorBrush x:Key="CriticalButtonBackgroundPressed"
+                                 Color="Black" />
+                <SolidColorBrush x:Key="CriticalButtonBackgroundDisabled"
+                                 Color="Gray" />
+
+                <SolidColorBrush x:Key="CriticalButtonBorderBrush"
+                                 Color="White" />
+                <SolidColorBrush x:Key="CriticalButtonBorderBrushPointerOver"
+                                 Color="White" />
+                <SolidColorBrush x:Key="CriticalButtonBorderBrushPressed"
+                                 Color="White" />
+                <SolidColorBrush x:Key="CriticalButtonBorderBrushDisabled"
+                                 Color="White" />
+
+                <SolidColorBrush x:Key="CriticalButtonForeground"
+                                 Color="White" />
+                <SolidColorBrush x:Key="CriticalButtonForegroundPointerOver"
+                                 Color="Black" />
+                <SolidColorBrush x:Key="CriticalButtonForegroundPressed"
+                                 Color="White" />
+                <SolidColorBrush x:Key="CriticalButtonForegroundDisabled"
+                                 Color="Black" />
+            </ResourceDictionary>
+
+        </ResourceDictionary.ThemeDictionaries>
+    
+    <Style x:Key="CriticalButtonStyle"
+           TargetType="Button">
+        <Setter Property="Foreground"
+                Value="{ThemeResource CriticalButtonForeground}" />
+        <Setter Property="Background"
+                Value="{ThemeResource SystemFillColorCriticalBrush}" />
```

Comment:
I think we should define a CriticalButtonBackground brush and have it point it to this for completeness sake.